widget/embedded/src/types/config.ts

```diff
@@ -209,4 +213,5 @@ export type WidgetConfig = {
   features?: Features;
   variant?: WidgetVariant;
   enableCentralizedSwappers?: boolean;
+  signerConfig?: SignerConfig;
```

Comment:
It's already in a `config` so adding a suffix is not needed. I mean you can rename `signerConfig` to `signers` or whatever makes more sense to you.